pkg/queue/request_metric_test.go

```diff
@@ -64,37 +64,36 @@ func TestRequestMetricHandler(t *testing.T) {
 	}
 }
 
-func TestRequestMetricHandler_PanickingHandler(t *testing.T) {
+func TestRequestMetricHandlerPanickingHandler(t *testing.T) {
 	baseHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		panic("no!")
 	})
 	r := &fakeStatsReporter{}
 	handler, err := NewRequestMetricHandler(baseHandler, r)
 	if err != nil {
-		t.Fatalf("failed to create handler: %v", err)
+		t.Fatalf("Failed to create handler: %v", err)
```

Comment:
What I have learned is that:
* A sentence of a log begins with Capital.
* A sentence of an error message begins with lower case.

Do `t.Fatalf` and something else belong to logs?

Reply:
testing errors are full sentences.
Only `error`s are snippets.